packages/components/src/input-control/index.js

```diff
@@ -0,0 +1,137 @@
+/**
+ * External dependencies
+ */
+import { noop } from 'lodash';
```

Comment:
I've seen this code pattern used in several places, but given that we can now use optional chaining, I think that instead of doing this, we could just let the functions be `undefined` by default. I'm just guessing (and could be completely wrong about this), but it seems like that would perform slightly better. To avoid calling functions that are `undefined`, we would use optional chaining like so:

```js
onBlur?.( event )`
```

Again, this is just an idea; it probably doesn't matter much either way.

Reply:
Interesting idea! I thought about it for a bit (considering optional chaining is a new thing)

From my experience in the React library/module space, I've found it smoother for callbacks to fire without guards. It kept the code tidier and there was less to think about.

That being said, optional chaining does tidy it up a lot.

I'm perhaps biased in that I still prefer `noop` callbacks.
I'm open to using optional chaining instead, if that's what others prefer 👍